clients/frama-c-client/log-creator/.gitignore

```diff
@@ -0,0 +1,3 @@
+LogCreatorIdentity.json
+Channel.json
```

Comment:
this file should be part of the repository please add it for creator and auditor